core/autocomplete/templating/AutocompleteTemplate.ts

```diff
@@ -458,12 +524,16 @@ export function getTemplateForModel(model: string): AutocompleteTemplate {
     return codegeexFimTemplate;
   }
 
+  if (lowerCaseModel.includes("granite")) {
+    if (/(granite[^0-9]*3\.([0-2]))|granite-3\.([0-2])/i.test(lowerCaseModel))
```

Comment:
I don't think the second alternative is needed - because of the [^0-9]* in the first alternative. And since lowerCaseModel is already lowercase, the i shouldn't be needed either.

Reply:
yea not sure what i was thinking there..